.evergreen/run-serverless-tests.sh

```diff
@@ -3,7 +3,6 @@
 source $DRIVERS_TOOLS/.evergreen/init-node-and-npm-env.sh
 
 if [ -z ${SERVERLESS+omitted} ]; then echo "SERVERLESS is unset" && exit 1; fi
-if [ -z ${SERVERLESS_URI+omitted} ]; then echo "SERVERLESS_URI is unset" && exit 1; fi
```

Comment:
This variable is never used in our tests because we use the load balanced URIs instead.